charts/base-cluster/values.yaml

```diff
@@ -443,7 +443,7 @@ rbac:
   accounts: {}
 
 backup:
-  enabled: true
+  enabled: false
```

Comment:
This is a breaking change, which is fine, you just need to change your commit message to include an `!`; `fix(base-cluster)!:`

And you should include the subarea in you message; `fix(base-cluster/backup)!:`